File: 1013.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');

// URI 1013 | O MAIOR

var values = input.split(' ')
var a = parseInt(values.shift())
var b = parseInt(values.shift())
var c = parseInt(values.shift())

var maiorAB = (a + b + Math.abs(a - b)) / 2

if (maiorAB > c) {
  console.log(maiorAB + " eh o maior")
} else {
  console.log(c + " eh o maior")
}

File: Python/1013.py
# BEE 1013 | O MAIOR

line = input().split(' ')
A = int(line[0])
B = int(line[1])
C = int(line[2])

maiorAB = (A + B + abs(A - B)) / 2

if maiorAB > C:
    print("%d eh o maior" % maiorAB)
else:
    print("%d eh o maior" % C)